pyproject.toml

```diff
@@ -1,3 +1,20 @@
+[project]
+name = "torch"
+requires-python = ">=3.9"
+license = {text = "BSD-3-Clause"}
+dynamic = [
+  "authors",
```

Comment:
These are filled from reading the logs / warnings from `uv sync`

Reply:
In a follow up PR I can migrate the static fields from setuptools to the pyproject.toml, setuptools will then automatically popuate them from the pyproject.toml